changelog/dmd.alias-instance-member.dd

```diff
@@ -0,0 +1,25 @@
+[next edition] Aliasing a member of a type *instance* is now an error
```

Comment:
I'm not sure what the procedure is for documenting editions changes, so I have kept a changelog entry for now. It could be documented in [spec/legacy](https://dlang.org/spec/legacy), like the [body change](#16382) already is.